apiary-metastore-events/kafka-metastore-events/kafka-metastore-listener/src/main/java/com/expediagroup/apiary/extensions/events/metastore/KafkaProducerProperty.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (C) 2018-2019 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.apiary.extensions.events.metastore;
```

Comment:
com.expediagroup.apiary.extensions.events.metastore ->  com.expediagroup.apiary.extensions.events.kafka.metastore;

Reply:
your comment at the top suggests `com.expediagroup.apiary.extensions.events.metastore.kafka.common`, so this should probably be `com.expediagroup.apiary.extensions.events.metastore.kafka`?

Reply:
Yeah, I guess, let's see what it looks like when we have the new proposed modules etc. and where the most logical place to put "kafka" in the package name is.